.github/workflows/node.js.yml

```diff
@@ -10,16 +10,49 @@ on:
         branches: [master]
 
 jobs:
+    clone_vector_skin:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Clone vector skin repository"
+              run: |
+                  git clone "https://gerrit.wikimedia.org/r/mediawiki/skins/Vector" skins/Vector
+
+    mediawiki_docker:
```

Comment:
I've confirmed that we need to create a `docker-compose.override.yml` file for linux users and do the steps. Please add those to the workflow as well.

It should be run before `docker-compose up -d`

Reply:
Noted @und3fined-v01d